tests/torch/requirements.txt

```diff
@@ -12,3 +12,4 @@ pyparsing<3.0
 
 # Required for search_buidling_blocks tests
 transformers
+datasets
```

Comment:
@ljaljushkin just added. :)

Reply:
@ljaljushkin is that the intent to not have a specific version for ```transformers``` and ```datasets```?

Reply:
No, I think we should define a specific version. For example, latest release of transformers - v4.23.1.